### ISSUE TITLE
Lava mobs aren't allowed to escape lava pools in the nether

### ISSUE THREAD

**Issue by Githubberclown** [Ladysnake/Requiem#239]
Okay i don't know if this is a bug or an unintended issue but its frankly game breaking and i think it NEEDS to be fixed, but the scenario: You just died in a nether fortress and you're a demon, you posess a wither skeleton or a zombie piglin and you accidentally mess up and fall into the lava lake below the fortress.

What happens is because undead aren't able to swim, right? What happens is because unlike being underwater where you would be able to climb up before you die from drowning, in the nether (YOU CANNOT ACTUALLY CLIMB UP BLOCKS, SO YOU CANNOT ESCAPE THE LAVA LAKE) So what happens is that when you try to climb up a block so you can make a staircase up to the top of the lava lake, the undead mob doesnt jump high enough to get on top of the block so im stuck in the lava lake, and i can't ever get out, and if im a wither skeleton if i use up too many bones i become a skeleton and i die in the lava, so that poses another risk because if i die i lose everything and potentially massive progress to getting cured.

I'd just like this to get fixed, make mobs able to climb up full blocks in the nether maybe? thanks, im not trashing your mod its just really annoying to be demon because falling in the lava is basically a game ender/forces creative mode if you don't wanna spend hours finding your way out of the lava lake.

**Comment by Githubberclown** [comment]
TLDR: When you fall in a lava lake, due to not being able to swim up out of the lake, you are forced to go up one block, but due to the fact that lava is thick and makes you move slower in every direction, you don't swim up far enough to reach a block higher than the one you're standing on, unless its a half slab.

Edit: One cannot simply jump up one block as an undead mob whilst inside lava lmao

**Comment by Pyrofab** [comment]
Fixed in 1.5.0.
